### ISSUE TITLE
Improve help text after challenge and when starting game

### ISSUE THREAD

**Issue by mtchd** [mtchd/connect4#5]
The following help text isn't super readable:

```
"They must respond with 'accept' or 'reject' or 'accept -token :your-emoji-here:'"
```

```
Available commands:
'$columnNumber'
'forfeit'
```

Better would be:

```
You can respond with:
accept
reject
accept :emoji:
```

```
Available commands:
column number (e.g. 1)
forfeit
```

![giphy (7)](https://user-images.githubusercontent.com/30946820/63237274-a585ed00-c284-11e9-96a2-341c47e8423a.gif)

**Comment by mtchd** [comment]
Fixed in 0.2.1!

**Comment by mtchd** [comment]
0.2.1 has now been pushed to prod, closing issue!
